openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/TestStreamProcessor.java

```diff
@@ -126,4 +126,5 @@ public void destroy() {
 		}
 		streamByUid.clear();
 	}
-}
+
+}
```

Comment:
Please add line break :)

Reply:
well if that is really important can you add a rule to checkstyle? I think I never touched this file it was a pure accident. But its now the 2nd commit I make just to change the line breaks.

Reply:
It should be now back to what it was initially.

Reply:
Might be good idea to add rule :)